include/rdma/fi_domain.h

```diff
@@ -115,6 +115,11 @@ struct fid_av {
  * Tracks registered memory regions, primarily for remote access,
  * but also for local access until we can remove that need.
  */
+
+#define FI_HMEM_HOST_ALLOC		(1ULL << 60)
+#define FI_HMEM_DEVICE_ONLY		(1ULL << 61)
+#define FI_HMEM_DATA_GDRCOPY_HANDLE	(1ULL << 62)
```

Comment:
The values need to start at 59 and go *down* (i.e. 58, 57, etc.).  The bit values 60-63 are reserved for provider specific use (or by libfabric for internal use).

Reply:
Got it. I was using a different endianess.